README.md

```diff
@@ -42,11 +43,11 @@ With ice_cube, you can specify (in increasing order of precedence):
 * Recurrence Times - To specifically include in a schedule
 * Exception Times - To specifically exclude from a schedule
 
-Example: Specifying a recurrence with an exception time:
+Example: Specifying a recurrence with an exception time (ActiveRecord:
```

Comment:
Oops. That was going to grow into a 'require' because 1.day etc only works with rails helpers but then I went back to take it out and missed one.